bazel_tools/haskell-cabal-cc-wrapper.patch

```diff
@@ -0,0 +1,49 @@
+diff --git a/haskell/cabal.bzl b/haskell/cabal.bzl
```

Comment:
I suppose that’s upstreameable as well?

Reply:
Yes, I haven't open a PR, yet. But, I'll mark it and others as upstreamable.